transforms/packaging/python/Makefile

```diff
@@ -0,0 +1,61 @@
+# Define the root of the local git clone for the common rules to be able
+# know where they are running from.
+REPOROOT=../../..
+# Include a library of common .transform.* targets which most
+# transforms should be able to reuse.  However, feel free
+# to override/redefine the rules below.
+
+# $(REPOROOT)/.make.versions file contains the versions
+
+include $(REPOROOT)/transforms/.make.transforms
+include ../.make.packaging
+
+PACKAGING_RUN_TIME=python
+
+
+#Excluded List
+# ./code/malware
+# ./universal/html2parquet
+# ./universal/profiler      # Missing implementation 
+# ./universal/fdedup 	    # Missing implementation
+#  code/repo_level_ordering # Missing implementation
+
+
+TRANSFORMS_NAMES = code/code_quality \
```

Comment:
We should build this list automatically by looking for the python directories in transforms/*/*